### PR TITLE
[powerpoint] (Presentation) Adding createPresentation section

### PR DESCRIPTION
**Pull request by AlexJerabek**
Adding a description of the new `createPresentation` method to the PowerPoint article. Also cleaning up a couple spacing warnings.

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[0600324](https://github.com/OfficeDev/office-js-docs-pr/pull/701/commits/0600324a417a7b6e3ba9c42d1973da3f9ac19e83)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[0600324](https://github.com/OfficeDev/office-js-docs-pr/pull/701/commits/0600324a417a7b6e3ba9c42d1973da3f9ac19e83)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[0600324](https://github.com/OfficeDev/office-js-docs-pr/pull/701/commits/0600324a417a7b6e3ba9c42d1973da3f9ac19e83)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-PPT/docs/powerpoint/powerpoint-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201901241942031777-701%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=prQlbFkedd%2FpkLDN1tZSCcz6CT9vwODp4xyG7%2FrQZ9I%3D&st=2019-01-24T19%3A38%3A17Z&se=2019-02-24T19%3A43%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/powerpoint/powerpoint-add-ins.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: PowerPoint add-ins
 description: ''
-ms.date: 10/16/2018
+ms.date: 01/24/2018
```
Comment (kbrandl) [inline docs/powerpoint/powerpoint-add-ins.md]: Year should be changed to 2019

**Comment by o365devx** [comment]
OPS Build status updates of commit _[2490f4c](https://github.com/OfficeDev/office-js-docs-pr/pull/701/commits/2490f4c099220da2cf504f3b629f901a8bef8774)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by AlexJerabek** [comment]
@kbrandl Thanks Kim. Updated with the current year.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[2490f4c](https://github.com/OfficeDev/office-js-docs-pr/pull/701/commits/2490f4c099220da2cf504f3b629f901a8bef8774)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[2490f4c](https://github.com/OfficeDev/office-js-docs-pr/pull/701/commits/2490f4c099220da2cf504f3b629f901a8bef8774)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/powerpoint/powerpoint-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-PPT/docs/powerpoint/powerpoint-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201901242149554793-701%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=f5hFMK5agBWgxo%2FALFpv6vaPyFAUzntvTJ4vqmamJhw%3D&st=2019-01-24T21%3A46%3A12Z&se=2019-02-24T21%3A51%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**